src/encoded/static/components/item-pages/MicroscopeConfigurationView.js

```diff
@@ -0,0 +1,588 @@
+                            });
+                        });
+                    },
+                    'POST',
+                    fallbackCallback,
+                    JSON.stringify(payload)
+                );
+            }
+        );
+
+    }
+
+    /**
+      * Cancel/hide modal popup
+      * @returns {void}
+    */
+    handleModalCancel() {
+        this.setState({ 'modal': null });
+    }
+
+    /**
+        * Copies current URL to clipbard.
+        * Sets the microscope display status to released if it isn't already.
+        *
+        * @returns {void}
+    */
+    handleStatusChange(statusToSet = 'released', evt) {
+        evt.preventDefault();
+
+        const { context, href } = this.props;
+        const mtc = this.getMicroscopyMetadataToolComponent();
+        const confTitle = context.title || context.display_title;
+
+        // If the view config has already been released, just copy the URL to the clipboard and return.
+        if (context.status === statusToSet) {
+            return;
+        }
+
+        if (!this.havePermissionToEdit()) {
+            throw new Error('No edit permissions.');
+        }
+
+        // PATCH `status: released` to current href, then in a callback, copy the URL to the clipboard.
+        this.setState(
+            { 'releaseLoading': true },
+            () => {
+                ajax.load(
+                    href,
+                    (resp) => {
+                        // Success! Generate an alert telling the user it's successful
+                        this.setState({ 'releaseLoading': false });
+                        Alerts.queue({
+                            'title': "Updated Status for " + confTitle,
+                            'message': (
+                                <p className="mb-02">
+                                    Changed Display status to <b>{statusToSet}</b>.
+                                    It may take some time for this edit to take effect.
+                                </p>
+                            ),
+                            'style': 'info'
+                        });
+                    },
+                    'PATCH',
+                    (resp) => {
+                        // Error callback
+                        this.setState({ 'releaseLoading': false });
+                        Alerts.queue({
+                            'title': "Failed to release display.",
+                            'message': "Sorry, can you try to share again?",
+                            'style': 'danger'
+                        });
+                    },
+                    JSON.stringify({
+                        'status': statusToSet
+                    })
+                );
+            }
+        );
+    }
+
+    /** @todo make into functional component */
+    statusChangeButton(){
+        const { session, context } = this.props;
+        const { saveLoading, cloneLoading, releaseLoading } = this.state;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['cloneLoading' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4271272)